x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -353,7 +421,110 @@ export const AlertsList: React.FunctionComponent = () => {
+      {!dissmissAlertErrors && alertsStatusesTotal[AlertExecutionStatusValues[2]] > 0 ? (
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiCallOut
+              color="danger"
+              size="s"
+              title={
+                <FormattedMessage
+                  id="xpack.triggersActionsUI.sections.alertsList.attentionBannerTitle"
+                  defaultMessage="There is {totalStausesError} {totalStausesError, plural, one {{singleTitle}} other {# {multipleTitle}}} with an error."
+                  values={{
+                    totalStausesError: alertsStatusesTotal[AlertExecutionStatusValues[2]],
+                    singleTitle: 'alert',
+                    multipleTitle: 'alerts',
+                  }}
+                />
+              }
+              iconType="alert"
+            >
+              <EuiButton
+                type="primary"
+                size="s"
+                color="danger"
+                onClick={() => setAlertStatusesFilter([AlertExecutionStatusValues[2]])}
+              >
+                <FormattedMessage
+                  id="xpack.triggersActionsUI.sections.alertsList.viewBunnerButtonLabel"
+                  defaultMessage="View"
+                />
+              </EuiButton>
+              <EuiButtonEmpty color="danger" onClick={() => setDissmissAlertErrors(true)}>
+                <FormattedMessage
+                  id="xpack.triggersActionsUI.sections.alertsList.dismissBunnerButtonLabel"
+                  defaultMessage="Dismiss"
+                />
+              </EuiButtonEmpty>
+            </EuiCallOut>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      ) : null}
+      <EuiSpacer size="m" />
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          <EuiText size="s" color="subdued">
+            <FormattedMessage
+              id="xpack.triggersActionsUI.sections.alertsList.totalItemsCountDescription"
+              defaultMessage="Showing: {pageSize} of {totalItemCount} alerts."
+              values={{ totalItemCount: alertsState.totalItemCount, pageSize: page.size }}
+            />
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiHealth color="primary">
+            <FormattedMessage
+              id="xpack.triggersActionsUI.sections.alertsList.totalStausesActiveDescription"
+              defaultMessage="Active: {totalStausesActive}"
+              values={{
+                totalStausesActive: alertsStatusesTotal[AlertExecutionStatusValues[1]],
+              }}
+            />
+          </EuiHealth>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiHealth color="danger">
+            <FormattedMessage
+              id="xpack.triggersActionsUI.sections.alertsList.totalStausesErrorDescription"
+              defaultMessage="Errors: {totalStausesError}"
+              values={{ totalStausesError: alertsStatusesTotal[AlertExecutionStatusValues[2]] }}
```

Comment:
nit: Would it make it clearer to define constants for these array index accessors?